src/layouts/index.js

```diff
@@ -95,6 +95,10 @@ const query = graphql`
     }
   }
 `
+const generateTitle = prefix => `${prefix} | Pride in London`
```

Comment:
To avoid duplication, extract this logic out into a common `Helmet` component. This could then be imported wherever needed. There could be a `defaultProps` assignment for the description `In June and July 2019...` section that crops up most frequently, yet leaves the possibility to override.

You could have something like:

```js
const PrideHelmet = ({ title, description }) => (
  <Helmet
    title={`${title} | Pride in London`}
    meta={[
      {
        name: 'description',
        content: `${title}. ${description}`,
      },
    ]}
  />
)
```